src/plugins/ResizeReorder/ReorderCell.js

```diff
@@ -92,17 +94,22 @@ class ReorderCell extends React.PureComponent {
 
     let style = {
       height: props.height,
-      width: props.width - BORDER_WIDTH,
+      width: props.width,
     };
 
+    if (!this.props.shouldUseLegacyComponents) {
+      if (left === undefined) style.display = 'none';
```

Comment:
Please add a comment for why you do this ?
and what the reason for left going undefined here ?

Reply:
Is it only the case with ReorderCell ? what about regular FixedDataTableCell ?

Reply:
This was occuring only in reorder cell because earlier I was setting left of style here only but I can directly use the styleDefault in which I will fix the left in FixedDataTableCell and then pass the whole styleDefault as prop to reorder cell .So left will be set according to the rendering of FixedDataTableCell component .I corrected this thing in the new commit in other way and dropped this hacky way .